frontend/src/metabase/admin/databases/containers/DatabaseEditApp.tsx

```diff
@@ -83,146 +71,121 @@ const mapStateToProps = (state: State) => {
-      if (addingNewDatabase) {
-        scheduleCallback(() => {
-          onChangeLocation(
-            `/admin/databases?created=true&createdDbId=${savedDB.id}`,
-          );
-        });
-      }
-    } catch (error) {
-      throw getSubmitError(error as DatabaseEditErrorType);
-    }
-  };
+    database?.name && [database?.name],
+  ]);
 
-  const autofocusFieldName = window.location.hash.slice(1);
+  const isPermissionModalOpened = !!location.query.created || false;
+  const onPermissionModalClose = () => replace(location.pathname);
 
   return (
-    <DatabaseEditRoot>
-      <Breadcrumbs className={CS.py4} crumbs={crumbs} />
-
-      <DatabaseEditMain>
-        <ErrorBoundary errorComponent={GenericError as ComponentType}>
-          <div>
-            <div className={CS.pt0}>
-              <LoadingAndErrorWrapper
-                loading={!database}
-                error={initializeError}
-              >
-                {editingExistingDatabase && database.is_attached_dwh ? (
-                  <div>{t`This database cannot be modified.`}</div>
-                ) : (
-                  <DatabaseEditContent>
-                    <DatabaseEditForm>
-                      <DatabaseForm
-                        initialValues={database}
-                        isAdvanced
-                        onSubmit={handleSubmit}
-                        setIsDirty={setIsDirty}
-                        autofocusFieldName={autofocusFieldName}
-                      />
-                    </DatabaseEditForm>
-                    <div>{addingNewDatabase && <DatabaseEditHelp />}</div>
-                  </DatabaseEditContent>
-                )}
-              </LoadingAndErrorWrapper>
-            </div>
-          </div>
-        </ErrorBoundary>
-
-        {editingExistingDatabase && !database.is_attached_dwh && (
-          <Sidebar
-            database={database}
-            isAdmin={isAdmin}
-            isModelPersistenceEnabled={isModelPersistenceEnabled}
-            updateDatabase={updateDatabase}
-            deleteDatabase={deleteDatabase}
-            dismissSyncSpinner={dismissSyncSpinner}
-          />
-        )}
-      </DatabaseEditMain>
-
-      <LeaveConfirmationModal
-        isEnabled={isDirty && !isCallbackScheduled}
-        route={route}
-      />
-    </DatabaseEditRoot>
+    <>
+      <ErrorBoundary errorComponent={GenericError as ComponentType}>
+        <Box w="100%" maw="64.25rem" mx="auto" px="2rem">
+          <Breadcrumbs className={CS.py4} crumbs={crumbs} />
+
+          <LoadingAndErrorWrapper loading={isLoading} error={initializeError}>
+            {database && (
+              <>
+                <ExistingDatabaseHeader database={database} />
+
+                <Divider mb={{ base: "1.5rem", sm: "3.25rem" }} />
+
+                <Flex
+                  direction="column"
+                  gap={{ base: "2rem", sm: "5.5rem" }}
+                  mb={{ base: "3rem", sm: "5.5rem" }}
```

Comment:
❤️   love the responsiveness

![Screenshot 2025-03-07 at 3 10 33 PM](https://github.com/user-attachments/assets/73a38e5a-3329-40eb-a128-34d346bfe296)